pyproject.toml

```diff
@@ -77,10 +77,10 @@ packages = [
     "resources/default/*",
     "resources/default/_app/*",
     "resources/default/_app/immutable/*/*",
-    "resources/wip/*",
-    "resources/wip/assets/*",
-    "resources/wip/assets/images/*",
-    "resources/wip/assets/config/*",
+    "resources/legacy/*",
+    "resources/legacy/assets/*",
+    "resources/legacy/assets/images/*",
+    "resources/legacy/assets/config/*",
```

Comment:
PLease, double-check. I believe `_app` and `_app/immutable` is for legacy, and new default should have `assets`, `assets/images`, and `assets/config`

Reply:
true, i did that wrong